cmd/admin-bucket-remote-add.go

```diff
@@ -50,6 +51,15 @@ var adminBucketRemoteAddFlags = []cli.Flag{
 		Name:  "bandwidth",
 		Usage: "Set bandwidth limit in bits per second (K,B,G,T for metric and Ki,Bi,Gi,Ti for IEC units)",
 	},
+	cli.BoolFlag{
+		Name:  "sync",
+		Usage: "enable synchronous replication for this target. Default is async",
+	},
+	cli.UintFlag{
+		Name:  "healthcheck-seconds",
+		Usage: "health check duration in seconds",
```

Comment:
What it does exactly ? health check timeout ?

Reply:
yes

Reply:
> frequency of health check to see if remote target is up

okay, so I guess you need to update this flag usage 'health check duration in seconds' is not that clear. like, 'Health check interval in seconds'